includes/class-wc-stripe-account.php

```diff
@@ -102,78 +102,11 @@ private function get_transient_key() {
 		return $key;
 	}
 
-	/**
-	 * Refetches account data and returns the fresh data.
-	 *
-	 * @return array Either the new account data or empty if unavailable.
-	 */
-	public function refresh_account_data() {
-		$this->clear_cache();
-		return $this->get_cached_account_data();
-	}
-
 	/**
 	 * Wipes the account data option.
 	 */
 	public function clear_cache() {
 		delete_transient( self::LIVE_ACCOUNT_OPTION );
 		delete_transient( self::TEST_ACCOUNT_OPTION );
 	}
-
-	/**
-	 * Indicates whether card payments are enabled for this (Stripe) account.
-	 *
-	 * @return bool True if account can accept card payments, false otherwise.
-	 */
-	private function are_payments_enabled( $account ) {
-		$capabilities = $account['capabilities'] ? $account['capabilities'] : [];
-
-		if ( empty( $capabilities ) ) {
-			return false;
-		}
-
-		return isset( $capabilities['card_payments'] ) && 'active' === $capabilities['card_payments'];
-	}
-
-	/**
-	 * Indicates if payouts are enabled for the (Stripe) account and if there is deposits schedule set.
-	 *
-	 * @return bool Returns 'false' if payouts aren't enabled for the (Stripe) account or of there is no
-	 * deposits schedule set.
-	 */
-	private function are_deposits_enabled( $account ) {
-		$are_payouts_enabled = $account['payouts_enabled'] || false;
-		$payout_settings     = isset( $account['settings']['payouts'] ) ? $account['settings']['payouts'] : [];
-
-		if ( ! $are_payouts_enabled || ! isset( $payout_settings['schedule']['interval'] ) ) {
-			return false;
-		}
-
-		return true;
-	}
-
-	/**
-	 * Gets the acoount's status from the acount data that is connected to this site.
-	 *
-	 * @return array Account status data or empty if failed to retrieve account data.
-	 */
-	public function get_account_status() {
```

Comment:
No longer needed, we're now just getting the data from the `useAccount()` hook in `wcstripe/data`